### DISCUSSION TITLE
Cannot install smi addon

### DISCUSSION THREAD

**Discussion by mryan43** [linkerd/linkerd2#9501] [Help]
Hello,

I have a working linkerd Installation on my local cluster and linkerd check works fine. When I try to enable the SNI plugin I get the following error : 

```
   $ linkerd smi install
   Cannot find Linkerd: invalid CIDR address: 
   Validate the install with: linkerd check
   exit status 1
```

The error message is really not helpful... what is the invalid address ? is it an empty string ? where is it configured / obtained from ?

**Comment by adleong** [comment]
Hi @mryan43.

I agree, that's a very strange error message but I can't reproduce it.  If I had to guess, it seems like something is wrong with your Linkerd config.  Can you run `linkerd check` to validate your installation?

**Comment by gbueno77** [comment]
I'm experiencing the same issue:

```
$ linkerd smi install
Cannot find Linkerd: invalid CIDR address: 
Validate the install with: linkerd check
exit status 1
```

$ linkerd smi version
v0.2.0

linkerd version
Client version: stable-2.13.1
Server version: stable-2.13.1
